### DISCUSSION TITLE
Music CD with multiple discs

### DISCUSSION THREAD

**Discussion by smw36164** [automatic-ripping-machine/automatic-ripping-machine#976] [Q&A]
Is there a variable within the abcde component where I can specify the CD number for those CD's that have multiple discs?
Something along the lines of OUTPUTFORMAT='${ARTISTFILE}/${ALBUMFILE}/${DISCNUM}/${TRACKNUM} - ${TRACKFILE}'

**Comment by 1337-server** [comment]
Not currently
